apps/website/app/(docs)/docs/roam/[slug]/page.tsx

```
@@ -0,0 +1,93 @@
+      slug,
+      directory: DIRECTORY,
+    });
+    const tableOfContents = await collectSections(contentHtml);
+
+    return (
+      <>
+        <div className="min-w-0 max-w-2xl flex-auto px-4 py-8 lg:max-w-none lg:pl-8 lg:pr-0 xl:px-16">
+          <article className="[&::-webkit-scrollbar]:hidden">
+            <DocsHeader title={data.title} />
+            <Prose>
+              <div dangerouslySetInnerHTML={{ __html: contentHtml }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)